src/memcmp.rs

```diff
@@ -1,100 +1,173 @@
 #![allow(dead_code)]
 
+use std::slice;
+
 #[inline]
-pub unsafe fn memcmp0(_: &[u8], _: &[u8]) -> bool {
+pub unsafe fn memcmp0(_: *const u8, _: *const u8, n: usize) -> bool {
```

Comment:
Is the new argument `n` really necessary? It should be implied by the function don't you think?

Reply:
It is so that it matches the signature of `memcmp` (which does need `n` as it now accepts a pointer rather than a reference) 😄